gcovr/formats/txt/write.py

```diff
@@ -54,8 +57,15 @@ def write_report(covdata: CovData, output_file: str, options: Options) -> None:
         fh.write("Directory: " + force_unix_separator(options.root) + "\n")
 
         fh.write("-" * LINE_WIDTH + "\n")
-        title_total = "Branches" if options.txt_use_branch_coverage else "Lines"
-        title_covered = "Taken" if options.txt_use_branch_coverage else "Exec"
+        title_total = "Lines"
+        title_covered = "Exec"
+        if options.txt_use_branch_coverage:
+            title_total = "Branches"
+            title_covered = "Taken"
+        elif options.show_decision:
+            title_total = "Decisions"
+            title_covered = "Taken"
```

Comment:
Unneeded initialization possible.
```suggestion
        if options.txt_use_branch_coverage:
            title_total = "Branches"
            title_covered = "Taken"
        elif options.show_decision:
            title_total = "Decisions"
            title_covered = "Taken"
        else:
            title_total = "Lines"
            title_covered = "Exec"
```